### PR TITLE
Added Requirement

### PR DESCRIPTION
**Pull request by VeeeneX**
Just to avoid #190

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -21,6 +22,12 @@ Table of Contents
 * [Development](#development)
 * [Testing](#testing)
 
+
+Requirements
+=============
+
+* Arc available on Jenkins
```
Comment (sectioneight) [inline readme.md]: Let's make this a bit more descriptive:

```markdown
* [Arcanist](https://github.com/phacility/arcanist) is installed on the Jenkins nodes where the tests will be run (e.g. not just the master) an the `arc` binary is in `$PATH` or configured explicitly in the global settings.
```

or something

Comment (VeeeneX) [inline readme.md]: Sure!

File: readme.md [hunk 2]
```diff
@@ -21,6 +22,12 @@ Table of Contents
 * [Development](#development)
 * [Testing](#testing)
 
+
+Requirements
+=============
+
+* [Arcanist](https://github.com/phacility/arcanist) is installed on the Jenkins nodes where the tests will be run (e.g. not just the master) an the `arc` binary is in `$PATH` or configured explicitly in the global settings.
```
Comment (sectioneight) [inline readme.md]: typo: **and** the ...

Also we should note that this is only required if you use the "Apply Phabricator Differential" part of the plugin. Some people just use the post-build action (e.g. to report data to harbormaster, which can be used either with staging areas or if you're running builds on diffusion commits). But we can clarify that later. For now just fix the typo then :shipit:

Thanks!

**Comment by sectioneight** [comment]
Thanks!

**Pull request merged**